### ISSUE TITLE
test accuracy

### ISSUE THREAD

**Issue by hkkevinhf** [junleen/RainNet#8]
Hi, I reproduce test performance (PSNR:35.8788,  MSE:44.5023 in my experiment) using your provided model weight (net_G.pth, net_G_last.pth).
However, the result seems different from your paper (PSNR:36.12, MSE: 40.29). Especially MSE is much higher than reported in the paper. Could you help me in solving this issue. Thanks.

**Comment by junleen** [comment]
Please check the version of PyTorch. Try with 1.7.0 or 1.8.1

**Comment by sj-li** [comment]
Hi, I face the same situation. When I evaluated with offered pre-trained model, the results are:

netG_latest:   PSNR: 35.7593  MSE: 42.2761
net_G_last:   PSNR: 34.7814  MSE: 54.4570
net_G:   PSNR:34.7762, MSE:53.6692

I test them with different PyTorch versions but the results are the same.
Do you know where is wrong?
Thanks~

**Comment by junleen** [comment]
Duplicate of #2

**Comment by sj-li** [comment]
Emmm, I use torch 1.7.0, but the results are still the same....

```
Package                 Version
----------------------- ---------
absl-py                 0.13.0
aiocontextvars          0.2.2
astor                   0.8.1
cached-property         1.5.2
cachetools              4.2.2
certifi                 2020.11.8
chardet                 3.0.4
charset-normalizer      2.0.4
chumpy                  0.70
contextvars             2.4
cycler                  0.10.0
Cython                  0.29.23
dataclasses             0.8
decorator               4.4.2
future                  0.18.2
gast                    0.4.0
google-auth             1.34.0
google-auth-oauthlib    0.4.5
google-pasta            0.2.0
grpcio                  1.39.0
h5py                    3.1.0
idna                    3.2
imageio                 2.9.0
immutables              0.14
importlib-metadata      3.1.0
Keras-Applications      1.0.8
Keras-Preprocessing     1.1.2
kiwisolver              1.3.1
loguru                  0.5.3
Markdown                3.3.4
matplotlib              2.2.3
mock                    4.0.2
mpi4py                  3.0.3
msgpack                 1.0.0
msgpack-numpy           0.4.7.1
multimethod             1.4
networkx                2.5.1
numpy                   1.19.5
oauthlib                3.1.1
opencv-contrib-python   4.1.0.25
opencv-python           4.5.3.56
Pillow                  8.0.1
pip                     21.2.2
protobuf                3.17.3
psutil                  5.7.3
pyasn1                  0.4.8
pyasn1-modules          0.2.8
pyparsing               2.4.7
python-dateutil         2.8.1
pytz                    2020.4
PyWavelets              1.1.1
PyYAML                  5.1.1
pyzmq                   20.0.0
requests                2.26.0
requests-oauthlib       1.3.0
rsa                     4.7.2
scikit-image            0.17.2
scipy                   1.5.4
setuptools              50.3.2
six                     1.15.0
tabulate                0.8.7
tensorboard             2.6.0
tensorboard-data-server 0.6.1
tensorboard-plugin-wit  1.8.0
tensorboardX            2.1
tensorpack              0.10.1
termcolor               1.1.0
tifffile                2020.9.3
torch                   1.7.0
torchvision             0.8.1
tqdm                    4.62.0
typing-extensions       3.10.0.0
urllib3                 1.26.6
vispy                   0.5.3
Werkzeug                2.0.1
wheel                   0.35.1
wrapt                   1.12.1
yacs                    0.1.8
zipp                    3.4.0
```
